src/praisonai/praisonai/cli/commands/run.py

```
@@ -640,6 +655,8 @@ class Args:
             print(result)
     
     except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)